src/utils/daos_hdlr.c

```diff
@@ -2505,11 +2508,14 @@ fs_copy_connect(struct file_dfs *src_file_dfs,
 				"container: %d\n", rc);
 			D_GOTO(err_src_root, rc);
 		}
-		rc = dfs_mount(fa->src_poh, fa->src_coh, O_RDWR,
-			       &src_file_dfs->dfs);
-		if (rc) {
-			fprintf(stderr, "dfs mount on source failed: %d\n", rc);
-			D_GOTO(err_src_dfs, rc);
+		if (is_posix_copy) {
```

Comment:
@dsikich Future improvement:
Similar to mpifileutils, it would be extensible to use something like `api in (DAOS, DFS)` instead of `is_posix_copy`.